js/src/modal.js

```diff
@@ -162,16 +161,17 @@ const Modal = (($) => {
       }
 
       if (this._isTransitioning) {
```

Comment:
@vanduynslagerp I think you should add a `!this._isShown` check here. If you don't and `hide()` is called when the modal is already hidden `_isTransitioning` is set to `true` and never reset to `false`.